test/rulesets/Base/fastmath_able.jl

```diff
@@ -49,7 +49,7 @@ const FASTABLE_AST = quote
     end
 
     @testset "Unary complex functions" begin
-        for x in (-4.1, 6.4)
+        for x in (-4.1, 6.4, 3 + im)
```

Comment:
this loop now tests we have `frule`s/`rrule`s for `angle` and `conj` on `Complex` numbers, which we don't yet, so we need to miss those out :)